internal/services/hdinsight/schema.go

```diff
@@ -112,6 +113,29 @@ func SchemaHDInsightsGateway() *pluginsdk.Schema {
 	}
 }
 
+func SchemaHDInsightsComputeIsolation() *pluginsdk.Schema {
+	return &pluginsdk.Schema{
+		Type:     pluginsdk.TypeList,
+		Optional: true,
+		MaxItems: 1,
+		Elem: &pluginsdk.Resource{
+			Schema: map[string]*schema.Schema{
+				"enable_compute_isolation": {
```

Comment:
we put enabled on the end
```suggestion
				"compute_isolation_enabled": {
```
but at that, as the block is compute isolation shouldn't this just be
```suggestion
				"compute_isolation_enabled": {
```

otherwise it would be `compute_isolation.compute_isolation_enabled` ?

Reply:
Will make the change.